diff --git a/dist/vite-plugin.js b/dist/vite-plugin.js
index c1c2b06e3c36bd1a35dee0e77bfad0c8ca4fed0d..399e24b0b4d65377fb4e331ecc755714cbefba23 100644
--- a/dist/vite-plugin.js
+++ b/dist/vite-plugin.js
@@ -37,7 +37,7 @@ function configPlugin(options) {
             if (name === VITE_ENVIRONMENT_NAMES.server) {
                 return {
                     build: {
-                        outDir: ".vercel/output/render.func",
+                        outDir: ".vercel/output/functions/render.func",
                         rolldownOptions: {
                             input: {
                                 [MAIN_INPUT]: ENTRYPOINT_VIRTUAL_MODULE,
