diff --git a/dist/vite-plugin.js b/dist/vite-plugin.js
index c1c2b06e3c36bd1a35dee0e77bfad0c8ca4fed0d..012790abc786d68f8b400eca5ac77112174fce61 100644
--- a/dist/vite-plugin.js
+++ b/dist/vite-plugin.js
@@ -30,14 +30,14 @@ function configPlugin(options) {
             if (name === VITE_ENVIRONMENT_NAMES.client) {
                 return {
                     build: {
-                        outDir: ".vercel/output/static",
+                        outDir: "dist",
                     },
                 };
             }
             if (name === VITE_ENVIRONMENT_NAMES.server) {
                 return {
                     build: {
-                        outDir: ".vercel/output/render.func",
+                        outDir: ".vercel/output/functions/__server.func",
                         rolldownOptions: {
                             input: {
                                 [MAIN_INPUT]: ENTRYPOINT_VIRTUAL_MODULE,
@@ -88,7 +88,21 @@ export function vercel({ handlerEntrypoint, requestLoggingLevel, ...userOptions
             userOptions,
             onBuildDone: async ({ output, serverEnvironment }) => {
                 const serverOutDir = join(serverEnvironment.config.root, serverEnvironment.config.build.outDir);
-                await writeFile(join(serverOutDir, "../config.json"), JSON.stringify({ version: 3 }), "utf-8");
+                await writeFile(join(serverOutDir, "../../config.json"), JSON.stringify({
+                    version: 3,
+                    framework: {
+                        version: `${PACKAGE_NAME}@${packageJson.version}`
+                    },
+                    routes: [
+                        {
+                            handle: "filesystem"
+                        },
+                        {
+                            src: "/.*",
+                            dest: "__server"
+                        }
+                    ]
+                }), "utf-8");
                 if (output === "static") {
                     // Clear server bundle
                     await rm(serverOutDir, {
